src/Tribe/Views/V2/Repository/Event_Period.php

```diff
@@ -344,7 +344,7 @@ public function not_in( $post_ids ) {
 	 * @return Tribe__Repository__Read_Interface
 	 */
 	public function parent( $post_id ) {
-		// TODO: Implement parent() method.
+		// @todo [BTRIA-595]: Implement parent() method.
```

Comment:
:warning: This comment is 42% valid code; is this commented out code?

<sub>hash: 5b99b9b945bdea4dbd65df15ba459003</sub>